documentation/database.md

```diff
@@ -0,0 +1,296 @@
+# Database
+
+> [!WARNING]
+> This documentation is meant to describe the state of the database. The reader should be aware that some undocumented changes may have been done after its last updates, and for this purpose should check the git history of this file.
+>
+> Also contributions are welcome! :heart:
+
+First of all, let's explicit two databases that Grist manages:
+1. The Home Database;
+2. The Document Database (aka the grist document);
```

Comment:
It's personal but I don't like the abbreviation in documentation, I feel it's not inclusive
I prefer more `also known as` than `aka`